zebra/interface.c

```diff
@@ -1156,6 +1156,91 @@ static const char *zebra_ziftype_2str(zebra_iftype_t zif_type)
 	}
 }
 
+/* Interface's brief information print out to vty interface. */
+static void ifs_dump_brief_vty(struct vty *vty, struct vrf *vrf)
+{
+	struct connected *connected;
+	struct listnode *node;
+	struct route_node *rn;
+	struct zebra_if *zebra_if;
+	struct prefix *p;
+	struct interface *ifp;
+	bool print_header = true;
+
+	FOR_ALL_INTERFACES (vrf, ifp) {
+		bool is_global_v6 = false;
+		char global_pfx[PREFIX_STRLEN] = {};
+		char ll_pfx[PREFIX_STRLEN] = {};
```

Comment:
While clang and gcc don't complain about this unless told to, I'm pretty sure it's not valid C, it needs to be `{0}`
```
vagrant@ubuntu-bionic ~> gcc -Wpedantic main.c
main.c: In function ‘main’:
main.c:51:20: warning: ISO C forbids empty initializer braces [-Wpedantic]
  char array1[20] = {};
```

Reply:
Done.